src/modules/acbs/dto/acbs-create-bundle-information-response.dto.ts

```diff
@@ -1,5 +1,6 @@
 import { AcbsBundleId } from '@ukef/helpers';
 
+// TODO APIM-128: Should we rename this to AcbsCreateBundleInformationResponseHeadersDto?
 export interface AcbsCreateBundleInformationResponseDto {
```

Comment:
It confused me that we call this a response DTO because ACBS doesn't return this as a response body - it returns it in the _headers_ instead. I was wondering what people thought about changing the name slightly to make that clearer

Reply:
Happy to rename it